.flake8

```diff
@@ -1,5 +1,5 @@
 [flake8]
-# E501 is not flexible enough, we're using B950 instead. Consistent with pytorch
-ignore = E402,E722,W503,W504,F821,E501
+ignore = E401,E402,E501,E722,W503,W504,F821,B006,B007,B008,B009
```

Comment:
I took the liberty to merge the `flake8` configs that were scattered across `.flake8`, `setup.cfg`, and `tox.ini`.

Reply:
Thank you for consolidating these. This is great!